Conan.VisualStudio/Services/ConanService.cs

```diff
@@ -117,43 +118,50 @@ private async Task InstallDependenciesAsync(ConanRunner conan, ConanProject proj
                     ConanBuildType build = _settingsService.GetConanBuild();
                     bool update = _settingsService.GetConanUpdate();
                    
-                    var process = await conan.Install(project, configuration, generator, build, update, _errorListService);
-
-                    string message = $"[Conan.VisualStudio] Calling process '{process.StartInfo.FileName}' " +
-                        $"with arguments '{process.StartInfo.Arguments}'";
+                    ProcessStartInfo process = conan.Install(project, configuration, generator, build, update, _errorListService);
 
+                    string message = $"[Conan.VisualStudio] Calling process '{process.FileName}' " +
+                                     $"with arguments '{process.Arguments}'";
                     Logger.Log(message);
                     await logStream.WriteLineAsync(message);
 
-                    using (var reader = process.StandardOutput)
+                    using (Process exeProcess = Process.Start(process))
                     {
-                        string line;
-                        while ((line = await reader.ReadLineAsync()) != null)
+                        int exitCode = await exeProcess.WaitForExitAsync();
+
+                        string output = await exeProcess.StandardOutput.ReadToEndAsync();
```

Comment:
so, this way we have all stdout, then all stderr, which might be hard to make a correlation, is it possible to capture them in order?

Reply:
Yes, we should tell the process to redirect `stderr` to `stdout` (o vice-versa). It is outside of my C# knowledge right now 😆 

Probably we need a helper function to call any `ProcessStartInfo` and do the right thing with all of this. Do you think this is needed for this PR or just open a new issue to fix it?

Reply:
let's do it now, as we're touching that code, let's make it correct at one shot.
shouldn't be too hard, something like that could be used as a base:
https://gist.github.com/rivy/030df34899e4305ea5837586bf1ff7a4